cpp/common/src/codingstandards/cpp/rules/unusedparameter/UnusedParameter.qll

```diff
@@ -32,10 +32,13 @@ query predicate lambdaExprParamHasAccess(Parameter parameter) {
 
 query predicate problems(UnusedParameter p, string message, Function f, string fName) {
   not isExcluded(p, getQuery()) and
-  (not isMaybeUnusedParameter(p) and
-   f = p.getFunction() and
-  // Virtual functions are covered by a different rule
-   not f.isVirtual()) and
+  (
+    not isMaybeUnusedParameter(p) and
+    f = p.getFunction() and
+    // Virtual functions are covered by a different rule
+    not f.isVirtual()
+  ) and
   message = "Unused parameter '" + p.getName() + "' for function $@." and
-  fName = f.getQualifiedName()
+  // fName = f.getQualifiedName()
+  fName = "TODO."
```

Comment:
@rvermeulen If the function is referred to by a lambda expression, then `f.getQualifiedName()` becomes a goal that cannot be satisfied, hence the entire row gets excluded. What would be a nice alternative to the formula?